.github/workflows/swift-toolchain.yml

```diff
@@ -2118,9 +2118,117 @@ jobs:
           symbolsFolder: ${{ github.workspace }}/BinaryCache
           searchPattern: '**/*.exe'
 
+  debugging_tools:
+    name: Debugging Tools
+    needs: [context, compilers, devtools, sdk]
+    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    env:
+      TOOLCHAIN_PATH: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            cpu: x86_64
+            triple: x86_64-unknown-windows-msvc
+
+          # TODO(thebrowsercompany/swift-build#125): Fix the arm64 build and uncomment this.
+          # - arch: arm64
+          #   cpu: aarch64
+          #   triple: aarch64-unknown-windows-msvc
+
+    steps:
+      - name: Download compilers
+        uses: actions/download-artifact@v4
+        with:
+          name: compilers-${{ matrix.arch }}
```

Comment:
Do we not need the amd64 compilers always as we are cross-compiling x64 -> arm64 and x64 -> x64?

Reply:
I believe so. Mind if I iterate on this while enabling the arm64 build in the next PR? I need to figure out how to cross compile with SPM and it'll be easier to iterate with a smaller diff base.

Reply:
But this change would give you a smaller diff because we know that this should be unconditional.

Reply:
Er, I guess I could just download them both even if the subsequent steps aren't using them yet. I'll try this out tomorrow.

Reply:
Done